Moonscraper Chart Editor/Assets/Scripts/Game/UI/Menus/SongPropertiesPanelController.cs

```diff
@@ -47,6 +49,25 @@ public class SongPropertiesPanelController : TabMenu
     TimeSpan customTime = new TimeSpan();
 
     static readonly string[] validAudioExtensions = { "ogg", "wav", "mp3", "opus" };
+    /// <summary>
+    /// null value indicates a valid filename that doesn't tie to a specific instrument (e.g. the song preview)
+    /// </summary>
+    static readonly Dictionary<string, Song.AudioInstrument?> validFilenames = new Dictionary<string, Song.AudioInstrument?>
```

Comment:
This needs more specific naming, as this list of audio files is specific to Rock Band/Clone Hero. Regular guitar hero custom songs are not specific to these audio names.